engine/standalone/engine/src/main/scala/pl/touk/nussknacker/engine/standalone/StandaloneProcessInterpreter.scala

```diff
@@ -55,7 +56,7 @@ object StandaloneProcessInterpreter {
       definitions,
       listeners,
       modelData.modelClassLoader.classLoader
-    ).andThen { compiledProcess =>
+    )(DefaultAsyncInterpretationValueDeterminer.DefaultValue).andThen { compiledProcess =>
```

Comment:
Why is it default here?

Reply:
I found out that `StandaloneMetaData` has no `useAsyncInterpretation` field so I assumed that configuration is not needed and it can be passed here whatever value, because it won't be read. Am I wrong? 

Maybe we should hardcode here `true` value because standalone is always executed asynchronous? But then why it is `allowLazyVars = true` hardcoded in `StandaloneMetaData`. Shouldn't be `false`?

Reply:
ok, I understand, it doesn't have any effect... probably it'll go away after removing lazy vals etc...